src/commands/Utility/quote.ts

```diff
@@ -142,6 +149,34 @@ async function addQuote(interaction: Discord.CommandInteraction, collection: Col
 		author = interaction.options.getUser("author", true),
 		url = interaction.options.getString("url", false)
 
+	let pictureUrl = interaction.options.getString("pictureUrl", false) ?? undefined
+
+	const urlSplit = url?.split("/")
+
+
+	if (urlSplit) {
```

Comment:
This should be declared in the const chain above. Also there are way too many empty lines here lol

Reply:
No, because it relies on the url variable (if its in the chain, the url constant doesnt "exist" yet). I could change it so it runs interaction.options.getString() but i dont see the need to call it twice

Reply:
I don’t know where you got that from but that’s just not true. Variables are declared in the order they appear in code. I believe TS even compiles this with multiple const keys and removes the comma chain

Reply:
oh, its actually true lol, i remember it breaking on me